<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionWindow</class>
 <widget class="QMainWindow" name="FunctionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#043c4c;
border-color: rgb(21, 45, 73);
color : rgb(189, 221, 235)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
    <item>
     <widget class="CustomGraphicsView" name="StagingArea">
      <property name="styleSheet">
       <string notr="true">color : rgb(4, 60, 76);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="SearchBar" name="searchBar"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color : rgb(189, 221, 235)</string>
        </property>
        <property name="text">
         <string>Functions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">color: rgb(189, 221, 235)
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menuBuild"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCompile">
   <property name="text">
    <string>Compile</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>./headers/CustomGraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>SearchBar</class>
   <extends>QLineEdit</extends>
   <header>./headers/SearchBar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
